models/users.js

```diff
@@ -304,6 +305,24 @@ const updateUserPicture = async (image, userId) => {
   }
 };
 
+/**
+ * Sets the numberOfHours field of passed UserId to userNumberOfHours
+ *
+ * @param userNumberOfHours { integer }: numberOfHours
+ * @param userId { string }: User id
+ */
+
+const updateNumberOfHours = async (userNumberOfHours, userId) => {
+  const userRef = userModel.doc(userId);
```

Comment:
you are trying to update this in user model but the no of hours is stored in user status model